src/WebJobs.Script/Config/RouteHandlingOptions.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Script.Configuration
+{
+    public class RouteHandlingOptions
+    {
+        private string _authenticationLevel;
+
+        /// <summary>
+        /// Mode determining how routes are mapped. "function" (default) maps httpTrigger functions
+        /// defined in function metadata. "all" creates a single catch-all route that handles all
+        /// requests and proxies it to the custom handler.
+        /// </summary>
+        public string Mode { get; set; } = "function";
```

Comment:
I am curious what value we get with making this explicit. Technically we can already identify the difference between default / "function" based purely on the presence of this config value or not.

I am not saying to remove it, I just want to understand the customer benefit from needing to set 2 config values instead of one.